idb/grpc/client.py

```diff
@@ -789,6 +791,48 @@ async def list_xctests(self) -> List[InstalledTestInfo]:
     async def send_events(self, events: Iterable[HIDEvent]) -> None:
         await self.hid(iterator_to_async_iterator(events))
 
+    @log_and_handle_exceptions
+    async def axtap(
+        self,
+        element: HIDElementType,
+        label: str,
+        x: float,
+        y: float,
+        duration: Optional[float] = None,
+        count: Optional[int] = 1,
+    ) -> None:
+        accessibilityInfo = await self.stub.accessibility_info(
+            AccessibilityInfoRequest(
+                point=None,
+                format=(AccessibilityInfoRequest.LEGACY),
+            )
+        )
+
+        elementFound = False
+
+        for item in json.loads(accessibilityInfo.json):
+            try:
+                axElement = HIDElementType(item["role_description"])
+                axLabel = item["AXLabel"]
+
+                elementFound = axElement == element and (
+                    label is None or axLabel == label
+                )
+
+                if elementFound:
+                    axframe = item["frame"]
+                    x += axframe["x"]
+                    y += axframe["y"]
+                    break
+            except:
+                pass
+
+        if elementFound:
+            for n in range(count):
+                await self.send_events(tap_to_events(x, y, duration))
+        else:
+            print("AXElement with AXLabel: ", label, " type: ", element, " not found.")
```

Comment:
We could log it, however let's not send it to `stdout` and preserve the current `tap` behavior.

Reply:
👍 Can change to logging instead of print.

Reply:
I would not preserve the current `tap` behavior because the coordinates provided are suppose to be relative to the accessibility element's frame and default to (1,1). It is probably better to just do nothing than send `tap` to unintended coordinates.